src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.h

```diff
@@ -0,0 +1,69 @@
+/*
+ ISC License
+
+ Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ */
+
+#ifndef _FLYBY_POINT_
+#define _FLYBY_POINT_
+
+#include <stdint.h>
+#include "architecture/utilities/bskLogging.h"
+#include "cMsgCInterface/AttRefMsg_C.h"
+#include "cMsgCInterface/NavTransMsg_C.h"
+#include "cMsgCInterface/NavAttMsg_C.h"
+#include "cMsgCInterface/EphemerisMsg_C.h"
+
+
+typedef enum flybyModel{
+    rectilinear = 0,
+    cwEquations = 1
+} FlybyModel;
+
+/*! @brief Top level structure for the sub-module routines. */
+typedef struct {
+
+    /* declare these quantities that always must be specified as flight software parameters */
+    double      deltaT;               //!< time between two subsequent reads of the filter information
```

Comment:
I would be more explicit with variable names. dtFilterData for example?

Reply:
Fixed it!